pkg/gofr/datasource/redis/redis.go

```diff
@@ -20,6 +20,8 @@ const (
 
 type Config struct {
 	HostName string
+	UserName string
```

Comment:
username is a single word, so this should be `Username` just like everywhere else in the framework

Reply:
done